Remove 14kb rule and combine `Congestion control` with `Slow start` (#28061)
mdn/content: files/en-us/web/performance/how_browsers_work/index.md

In our example above, the request is definitely less than 14KB, but the linked resources aren't requested until the browser encounters the links during parsing, described below.

### Congestion control / TCP slow start

> TCP packets are splitting into segments during transmission. Since TCP guarantees the sequence of packets, the server must receive the acknowledgment from the client in the form of an ACK packet after sending a certain number of segments.

Reviewer: ```suggestion
TCP packets are split into segments during transmission. Because TCP guarantees the sequence of packets, the server must receive an acknowledgment from the client in the form of an ACK packet after sending a certain number of segments.
```

> If server waits for an ACK after each segment, this will result in frequent ACKs from the client and may increase transmission time even in case of an low load network.

Reviewer: ```suggestion
If the server waits for an ACK after each segment, that will result in frequent ACKs from the client and may increase transmission time, even in the case of a low-load network.
```

> On the other hand, sending too many segments at once can lead to the fact that in a busy network the client will not be able to receive them and respond with ACK for a long time, and the server will have to repeat sending several times.

Reviewer: ```suggestion
On the other hand, sending too many segments at once can lead to the problem that in a busy network the client will not be able to receive the segments and will just keep responding with ACKs for a long time, and the server will have to keep re-sending the segments.
```

> In order to balance the number of transmitted segments, there is a {{glossary('TCP slow start')}}, an algorithm that gradually increases the amount of transmitted data until the maximum network bandwidth can be determined. And it reduces the amount of transmitted data in case of high network load.

Reviewer: ```suggestion
In order to balance the number of transmitted segments, the {{glossary('TCP slow start')}} algorithm is used to gradually increase the amount of transmitted data until the maximum network bandwidth can be determined, and to reduce the amount of transmitted data in case of high network load.
```

> The number of segments to be transmitted is specified by the Congestion window (CWND) which can be initialized to 1, 2, 4, or 10 MSS (MSS is 1500 bytes over the Ethernet protocol). This is the number of bytes to send, upon receipt of which the client must send an ACK.

Reviewer: ```suggestion
The number of segments to be transmitted is controlled by the value of the congestion window (CWND), which can be initialized to 1, 2, 4, or 10 MSS (MSS is 1500 bytes over the Ethernet protocol). That value is the number of bytes to send, upon receipt of which the client must send an ACK.
```

> If an ACK is received, then CWND will be increased by 2 times and the next time the server will be able to send more segments. If not received, then the CWND value will decrease by 2 times, thus achieving a balance between sending too many segments and too few.

Reviewer: ```suggestion
If an ACK is received, then the CWND value will be doubled, and so the server will be able to send more segments the next time. If instead no ACK is received, then the CWND value will be halved. That mechanism thus achieves a balance between sending too many segments, and sending too few.
```
